### ISSUE TITLE
It seems that overfitting problem still exists in your network

### ISSUE THREAD

**Issue by ZM-J** [clguo/SA-UNet#10]
When I use your code to reproduce results in your paper, overfitting still exists:
![gg](https://user-images.githubusercontent.com/21689514/99353964-868b6600-28e0-11eb-8748-da00f84b0014.png)
and when I use your data augmentation and flipping code to generate augmented dataset, and then use it to train the network, there is a more severe overfitting problem: after several epochs, training loss keeps decreasing, but validation accuracy keeps increasing.
Could you tell me why

**Comment by clguo** [comment]
I’m preparing for the graduation exam, sorry, I don’t have time to deal with code problems now

**Comment by clguo** [comment]
> When I use your code to reproduce results in your paper, overfitting still exists:
> ![gg](https://user-images.githubusercontent.com/21689514/99353964-868b6600-28e0-11eb-8748-da00f84b0014.png)
> and when I use your data augmentation and flipping code to generate augmented dataset, and then use it to train the network, there is a more severe overfitting problem: after several epochs, training loss keeps decreasing (even to 0), but validation accuracy keeps increasing.
> I only use DRIVE dataset to train and validate your network.
> Could you tell me why

I’m preparing for the graduation exam, sorry, I don’t have time to deal with code problems now
